abb_irb6640_support/package.xml

```diff
@@ -0,0 +1,48 @@
+<package>
+  <name>abb_irb6640_support</name>
+  <version>1.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 6640 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 6640 manipulators. This currently includes the
+      IRB 6640-185/2.8m (6640-185) only.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in the
+      <a href="http://www.abbrobots.co.uk/en/3HAC028284-en.pdfgit s">ABB IRB 6640
```

Comment:
Yea, I saw that last night, but I figured their web server was down. I just tried it and it is still down and not sure what to do, it was the only place I was able to locate the manual.